content/2024/chinese/_index.md

```diff
@@ -35,9 +37,29 @@ The Chinese version of this meeting will be provided by [Jakarta EE China Commun
     {{</ grid/div >}}
 {{</ grid/section-container >}}
 
-<!-- Committee section -->
+
+{{< grid/section-container id="sponsors" class="backdrop">}}
+  {{< events/sponsors event="chinese" year="2024" title="Online Lucky Draw" displayBecomeSponsor="false" >}}
+{{</ grid/section-container >}}
+
+<!-- Add agenda using legacy CSS -->
+{{< grid/section-container class="alt" id="plan-of-the-day" containerClass="backdrop padding-top-40" >}}
+  {{< grid/div class="container" isMarkdown="false">}}
+    {{< events/agenda event="chinese" year="2024" >}}
+  {{</ grid/div >}}
+{{</ grid/section-container >}}
+
+<!-- Add speakers section -->
+{{< grid/section-container id="speakers" class="speaker-container text-center">}}
+  {{< grid/div class="padding-top-40" isMarkdown="false">}}
+    {{< events/user_display year="2024" event="chinese" source="speakers" subpage="speakers" useCarousel="false" imageRoot="/images/speakers/" title="演讲人" headerClass="text-left" />}}
```

Comment:
I noticed the `title` parameter is Chinese here, but this is the English _index.md file.

Reply:
Done.